src/rabbit_core_ff.erl

```diff
@@ -95,3 +103,19 @@ remove_explicit_default_bindings(FeatureName, Queues) ->
     [rabbit_binding:remove_default_exchange_binding_rows_of(Q)
      || Q <- Queues],
     ok.
+
+
+%% -------------------------------------------------------------------
+%% Virtual host metadata.
+%% -------------------------------------------------------------------
+
+virtual_host_metadata_migration(_FeatureName, _FeatureProps, enable) ->
+  Tab = rabbit_vhost,
+  rabbit_table:wait([Tab]),
+  Fun = fun vhost:upgrade/1,
```

Comment:
We must be explicit with the target version of the record (in this migration function, we only know and care about version 2), so please use `vhost:upgrade_to(vhost_v2, Vhost)`. See `migrate_to_amqqueue_with_type()` for an example.